pyproject.toml

```diff
@@ -0,0 +1,72 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "soundata"
+dynamic = ["version"]
+description = "Python library for loading and working with sound datasets."
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.7"
+authors = [
+    {name = "Magdalena Fuentes", email = "mfuentes@nyu.edu"},
+]
+maintainers = [
+    {name = "Magdalena Fuentes", email = "mfuentes@nyu.edu"},
+]
+keywords = ["sound", "dataset loader", "audio"]
+classifiers=[
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "Topic :: Multimedia :: Sound/Audio :: Analysis",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10"
+]
+dependencies = [
+    "librosa>=0.10.0",
+    "numpy>=1.21.6",
+    "pandas>=1.3.5",
+    "tqdm>=4.65.0",
+    "jams>=0.3.4",
+    "flake8>=5.0.4",
+    "mypy>=0.982",
```

Comment:
`mypy` is not imported by the package, but only mentioned in `docs/source/contributing.rst`, e.g.
```rst
python -m mypy soundata --ignore-missing-imports --allow-subclassing-any
```
So I guess this can be moved to optional dependencies.